vllm/worker/habana_model_runner.py

```diff
@@ -494,6 +504,12 @@ def _setup_buckets(self) -> None:
         self.prompt_buckets = warmup_buckets(self.prompt_bs_bucket_cfg,
                                              self.prompt_seq_bucket_cfg)
 
+        if self.lora_config:
```

Comment:
@hlahkar Comment on why this check is needed.

Reply:
This is done to ensure the buckets for warmup are within the self.max_num_batched_tokens limit; as the lora layers cannot handle beyond this length. If @kzawora-intel feels this limitation is a generic limit for VLLM; we can remove the check and make the code generic.